### PR TITLE
[one-cmds] Revisit 'run'

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit implements running of 'one-cmd' command with a user set environment.

Draft: https://github.com/Samsung/ONE/pull/11179
Related: https://github.com/Samsung/ONE/issues/11146

ONE-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

File: compiler/one-cmds/onelib/utils.py [hunk 1]
```diff
@@ -134,15 +134,16 @@ def safemain(main, mainpath):
         sys.exit(255)
 
 
-def run(cmd, err_prefix=None, logfile=None):
+def run(cmd, err_prefix=None, logfile=None, env=None):
```
Comment (jinevening) [inline compiler/one-cmds/onelib/utils.py:137]: This change is to turn on logs for circle_mpqsolver.

Other tools control logs using `--verbose` option. Do you have any reason for directly passing env variables?

Comment (stamalakhov) [reply compiler/one-cmds/onelib/utils.py:137]: > This change is to turn on logs for circle_mpqsolver.
> 
> Other tools control logs using `--verbose` option. Do you have any reason for directly passing env variables?

@jinevening 
Yes. It's just for logging. Would it be better to implement logging by `verbose` option?

Comment (jinevening) [reply compiler/one-cmds/onelib/utils.py:137]: Yes. It would be better to keep the interface consistent.

Comment (stamalakhov) [reply compiler/one-cmds/onelib/utils.py:137]: > Yes. It would be better to keep the interface consistent.

Ok. Thank you.

**Pull request closed without merging**